x-pack/plugins/monitoring/server/routes/api/v1/_health/errors_helpers/errors_query.ts

```diff
@@ -47,7 +47,7 @@ export const errorsQuery = ({
               },
               {
                 range: {
-                  timestamp: {
+                  '@timestamp': {
```

Comment:
did `timestamp` ever work?

Reply:
It works except for `enterprisesearch` (added in this change) which does not have a `timestamp` alias. Since every data streams/metricbeat module exposes the `@timestamp` ecs field it made sense to update